### PR TITLE
ENH: Improve CI.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Improve the module's CI:
- Add Travis and AppVeyor CI configuration `*.yml` files.
- Rename and move the Circle CI `*.yml` file to the `.circleci` folder to
  conform to the `ITKModuleTemplate` skeleton.
- Add the Python setup `setup.py` file
- Add the Python wrapping `*.wrap` files.
- Add the CI badges to the `README.rst` file.
- Take advantage of the commit to improve the `README.rst` file.

### CODE REVIEW

File: wrapping/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+itk_wrap_module(PhaseSymmetryFilter)
```
Comment (dzenanz) [inline wrapping/CMakeLists.txt]: `PhaseSymmetryFilter` - copy-paste error?

File: wrapping/itkRLERegionOfInterestImageFilter.wrap [hunk 2]
```diff
@@ -0,0 +1,3 @@
+itk_wrap_class("itk::RLERegionOfInterestImageFilter" POINTER)
```
Comment (dzenanz) [inline wrapping/itkRLERegionOfInterestImageFilter.wrap]: The file is called `RLERegionOfInterestImageFilter`, but the class is called `RegionOfInterestImageFilter` - no RLE prefix, it is just a specialization of regular `RegionOfInterestImageFilter`.

Comment (jhlegarreta) [reply wrapping/itkRLERegionOfInterestImageFilter.wrap]: 👍 Changed in e114d83.

I wonder whether two classes having the same name is correct can lead to some confusion, though.

Comment (dzenanz) [reply wrapping/itkRLERegionOfInterestImageFilter.wrap]: **RoI** filters must have the same name. `itk::RLEImage` class was designed to be a drop-in replacement for `itk::Image`.

**Pull request merged**